hathor/indexes/memory_tx_group_index.py

```diff
@@ -0,0 +1,76 @@
+# Copyright 2021 Hathor Labs
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from abc import abstractmethod
+from collections import defaultdict
+from typing import DefaultDict, Iterable, Set, Sized, TypeVar
+
+from structlog import get_logger
+
+from hathor.indexes.tx_group_index import TxGroupIndex
+from hathor.transaction import BaseTransaction
+from hathor.util import not_none
+
+logger = get_logger()
+
+KT = TypeVar('KT', bound=Sized)
+
+
+class MemoryTxGroupIndex(TxGroupIndex[KT]):
+    """ Index of inputs/outputs by address
```

Comment:
It's not address anymore. Some methods of this class are also outdated.

Reply:
Fixed. Thanks!